StardewArchipelago/GameModifications/Modded/JunimoShopGenerator.cs

```diff
@@ -465,40 +503,39 @@ private void AddJunimoDistantLandsStock(Dictionary<ISalable, int[]> stock)
                 return;
             }
 
-            var voidMintIdentifier = "1 2 2 3 2/spring summer fall/109/"; //Done as modded seeds have variable ID but use ID in dictionary
-            var vileAncientIdentifier = "2 7 7 7 5/spring summer fall/108/";
-            var cropList = Game1.content.Load<Dictionary<int, string>>("Data\\Crops");
-            var voidMintSeeds = cropList.FirstOrDefault(x => x.Value.Contains(voidMintIdentifier)).Key;
-            var vileAncientFruitSeeds = cropList.FirstOrDefault(x => x.Value.Contains(vileAncientIdentifier)).Key;
+            var voidMintSeeds = _stardewItemManager.GetItemByName("Void Mint Seeds").Id;
+            var vileAncientFruitSeeds = _stardewItemManager.GetItemByName("Vile Ancient Fruit Seeds").Id;
             AddToJunimoStock(stock, voidMintSeeds, "Yellow", true);
             AddToJunimoStock(stock, vileAncientFruitSeeds, "Yellow", true);
         }
 
-        private static int Gcd(int val1, int val2) //Seemingly no basic method outside of BigInteger?
+        public int ValueOfOneItemWithWeight(int[] offerRatio, double weight)
+        {
+            return (int) Math.Pow(offerRatio[1] / offerRatio[0], weight);
+        }
+
+        private static int GreatestCommonDivisor(int firstValue, int secondValue) //Seemingly no basic method outside of BigInteger?
         {
-            var a = Math.Max(val1, val2);
-            var b = Math.Min(val1, val2);
-            var r = a % b;
-            if ( r == 0)
+            var largestValue = Math.Max(firstValue, secondValue);
+            var lowestValue = Math.Min(firstValue, secondValue);
+            var remainder = largestValue % lowestValue;
+            if ( remainder == 0)
             {
-                if (val1 > val2)
-                {
-                    return val2;
-                }
-                else
+                if (firstValue > secondValue)
```

Comment:
I can now tell for sure what I felt was wrong the first time. Here, instead of checking which one was biggest, just return "lowestValue" :)